tfx_bsl/public/proto/model_spec.proto

```diff
@@ -79,4 +79,23 @@ message AIPlatformPredictionModelSpec {
   // The name of the model's version.
   // If not specified, version won't be attached to the request.
   string version_name = 3;
+
+  // Optional.
+  // Configures how examples are converted to API requests.
+  // If not specified, example features are directly converted to json dicts according to feature type.
+  oneof example_config {
+    // Use AIPlatformPredictionSerializationConfig if the model accepts a single serialized example as input.
+    AIPlatformPredictionSerializationConfig serialization_config = 4;
+  }
 }
+
+// An example configuration type where examples are serialized and passed to an input tensor that accepts a single string.
+// API requests will contain the serialized example as a base64 encoded string.
+//
+// For example:
+// instances = [
+//  {'b64': '<base64 encoded serialized example 1>'},
+//  {'b64': '<base64 encoded serialized example 2>'},
+//  ...
+// ]
+message AIPlatformPredictionSerializationConfig {}
```

Comment:
[Dependent on the answer to my question to @rose-rong-liu 's previous comment]

If we don't need input name, I wonder whether we should keep this empty message. I would suggest just using a bool in example_config to indicate whether to turn on this feature.

Reply:
Good suggestion. We can also use the boolean in oneof.

Reply:
What is the canonical way to handle this? If the proto contains:
```proto
oneof example_config {
    bool use_serialization_config = 4;
}
```

Should user code check if a config proto contains the `use_serialization_config` field _and_ it is True?

I.e this would serialize examples:
```py
inference_spec_type = InferenceSpecType(
    ai_platform_prediction_model_spec=AIPlatformPredictionModelSpec(
        project_id='test_project',
        model_name='test_model',
        version_name='test_version',
        use_serialization_config=True
    )
)
```
But these would not:
```py
inference_spec_type = InferenceSpecType(
    ai_platform_prediction_model_spec=AIPlatformPredictionModelSpec(
        project_id='test_project',
        model_name='test_model',
        version_name='test_version',
        use_serialization_config=False
    )
)
```
```py
inference_spec_type = InferenceSpecType(
    ai_platform_prediction_model_spec=AIPlatformPredictionModelSpec(
        project_id='test_project',
        model_name='test_model',
        version_name='test_version'
    )
)
```

Reply:
Default value of a bool in proto is False. We only serialize if the field is True.